primaryHandler.go

```diff
@@ -331,20 +342,86 @@ func authChain(basicAuth []string, jwtVal JWTValidator, capabilityCheck Capabili
 		options = append(options, basculehttp.WithTokenFactory("Basic", basculehttp.BasicTokenFactory(basicAllowed)))
 	}
 
-	if jwtVal.Keys.URI != "" {
-		resolver, err := jwtVal.Keys.NewResolver()
-		if err != nil {
-			return alice.Chain{}, emperror.With(err, "failed to create resolver")
-		}
+	var jwtVal JWTValidator
+	// Get jwt configuration, including clortho's configuration
+	v.UnmarshalKey("jwtValidator", &jwtVal)
```

Comment:
Move the configuration fetching out & into the main call so viper isn't passed in.

Reply:
done